src/Appwrite/Platform/Workers/Mails.php

```diff
@@ -95,7 +103,21 @@ public function action(Message $message, Registry $register, Log $log): void
         $mail->addAddress($recipient, $name);
         $mail->Subject = $subject;
         $mail->Body = $body;
-        $mail->AltBody = \strip_tags($body);
+
+        $mail->AltBody = $body;
+        $mail->AltBody = preg_replace('/<style\b[^>]*>(.*?)<\/style>/is', '', $mail->AltBody);
```

Comment:
why are we doing this

Reply:
Because plaintext email has styles in it. This prevents it. It must have been been problem for a long time but we never noticed because we didn't check plaintext of the email.